core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/function/Filter.java

```diff
@@ -39,10 +39,10 @@
  * For more complex queries, a {@link Map} of groups to {@link ElementFilter}s can also be provided to either
  * {@link  uk.gov.gchq.gaffer.data.element.Edge}s or {@link uk.gov.gchq.gaffer.data.element.Entity}s.
  */
-public class Filter implements
+public class Filter implements Function,
         Operation,
```

Comment:
This is a redundant interface implementation, since the InputOutput<?> interface extends Operation.